package.json

```diff
@@ -5,12 +5,16 @@
   "main": "index.mjs",
   "scripts": {
     "start": "forever start index.mjs -p",
-    "dev": "node index.mjs",
+    "stop": "killall -SIGINT webaverse || true",
```

Comment:
on first run of `watch` there's no instance of webaverse running , and so the `restart` command would fail.  This allows us to restart regardless of if there's an instance running already or not, while still showing the output from `killall`.